test/unit/model/consent.test.ts

```diff
@@ -60,6 +60,27 @@ const completeConsent: Consent = {
   credentialPayload: 'dwuduwd&e2idjoj0w'
 }
 
+const consentWithOnlyUpdateFields: Consent = {
+  id: '1234',
+  credentialId: '123',
+  credentialType: 'FIDO',
+  credentialStatus: 'PENDING',
+  credentialChallenge: 'xyhdushsoa82w92mzs',
+  credentialPayload: 'dwuduwd&e2idjoj0w'
+}
+
+const conflictingConsent: Consent = {
+  id: '1234',
+  // Conflicting initiatorId and participantId
```

Comment:
does it mean that `conflictingConset` has the same `id` as `completeConsent` but different `initiatorId` and `participantId` ? Please explain how to understand the `conflict`.

Reply:
Yes.